src/Components/SmallChart/SmallChart.module.css

```
@@ -0,0 +1,28 @@
+.container {
+  display: grid;
+  border-radius: 1rem;
+  width: 40%;
+  height: max-content;
+  filter: drop-shadow(0 0 0.7rem grey);
+  background-color: var(--primary-bg-color);
+  padding-left: 0.1rem;
+  padding-right: 0.1rem;
```

It's not necessary to change it but I personally prefer this:

```suggestion
  padding: 0 0.1rem;
```